x-pack/plugins/session_view/public/components/DetailPanelListItem/index.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState, ReactNode } from 'react';
+import { EuiText } from '@elastic/eui';
+import { CSSObject } from '@emotion/react';
+import { useStyles } from './styles';
+
+interface DetailPanelListItemDeps {
+  children: ReactNode;
+  copy?: ReactNode;
+  display?: string;
+}
+
+interface EuiTextProps {
```

Comment:
it will be more maintainable if you extend the `EuiTextProps` from the `eui` itself:

```js
// ...
import { EuiText, EuiTextProps } from '@elastic/eui';
// ...
// extending adding css
interface EuiTextPropsCss extends EuiTextProps {
  css: CSSObject;
}
//...
// using it:

const props: EuiTextPropsCss = {
    size: 's',
    css: !!copy ? styles.copiableItem : styles.item,
};
//...
```

then if EuiTextProps API changes in the future, like if they add more sizes, the code won't break.